### ISSUE TITLE
_react2.proptypes.string undefined is not an object

### ISSUE THREAD

**Issue by yokesharun** [sbycrosz/react-native-credit-card-input#87]
react: 16.0.0-beta.5
react-native: 0.49.2
react-native-credit-card-input: ^0.3.3

i followed this documentation : [https://github.com/sbycrosz/react-native-credit-card-input](https://github.com/sbycrosz/react-native-credit-card-input)

Error  : _react2.proptypes.string undefined is not an object

**Comment by Andreyco** [comment]
I created PR #88 
For time being...
```
npm install https://github.com/Andreyco/react-native-credit-card-input.git
// or
yarn add https://github.com/Andreyco/react-native-credit-card-input.git
```

**Comment by Andreyco** [comment]
😩 there is so much mess under the hood

**Comment by yokesharun** [comment]
@Andreyco Thanks 👍

**Comment by Jamongkad** [comment]
Thanks @Andreyco!

**Comment by wtglover** [comment]
@sbycrosz @Andreyco Is anyone still looking at PRs for this? Project feels pretty dead, given the breaking changes with RN v0.50. Had to go in and fix the PropTypes and ImageBackground issues myself. Would love to submit a PR but can't even get access to push my branch.

**Comment by Jamongkad** [comment]
@wtglover can you please share the location of your work similar to what @Andreyco has done?

**Comment by sbycrosz** [comment]
Should be fixed in 4.1

**Comment by chenbekor** [comment]
@sbycrosz I don't think it is included in 4.1 can you please confirm? (especially  - the ImageBackground issue...)
